.github/workflows/sdk-regression.yml

```diff
@@ -0,0 +1,38 @@
+on:
+  pull_request:
+    types: [opened]
+  issue_comment:
+    types: [created]
+
+jobs:
+  check-regression:
+    name: Check if needs to run sdk regression
+    runs-on: ubuntu-latest
+    outputs:
+      run-regression: ${{steps.regcheck.outputs.triggered}}
+    steps:
+      - uses: khan/pull-request-comment-trigger@master
+        name: Check if regression
+        if: github.event_name == 'pull_request'
+        id: regcheck
+        with:
+          trigger: 'RUN_SDK_REGRESSION'
+  sdk-regression:
```

Comment:
do we need a separate `job` for this, can this be indented under `regression:` job - easier to navigate and debug?

Reply:
Won't it be easier to navigate debug if we have a separate job? We'll be able to see when it triggered and when it did not without getting into the logs